kim_commands.rst

```diff
@@ -945,84 +931,76 @@ The syntax for the *kim\_property* command is as follows:
+*key\_name\_value* argument.  These values may either be scalars or arrays,
+as stipulated in the property definition.
 
-For example,
+.. note::
+
+    Each map assigned to a *key\_name* must contain the *key\_name\_key*
+    "source-value" and an associated *key\_name\_value* of the appropriate
+    type (as defined in the relevant property definition).  For keys that are
+    defined as having physical units in their property definition, the
+    "source-unit" *key\_name\_key* must also be given a string value recognized
+    by `GNU units <https://www.gnu.org/software/units/>`_.
+
+Once a *kim\_property create* command has been given to instantiate a property
+instance, maps associated with the property's keys can be edited using the
+*kim\_property modify* command.  In using this command, the special keyword
+"key" should be given, followed by the property key name and the key-value pair
+in the map associated with that key that is to be set.  For example, the
+`atomic-mass <https://openkim.org/properties/show/2016-05-11/brunnels@noreply.openkim.org/atomic-mass>`_
+property definition consists of two property keys named "mass" and "species."
+An instance of this property could be created like so:
 
 .. parsed-literal::
 
+   kim_property create 1 atomic-mass
    kim_property modify 1 key species source-value Al
    kim_property modify 1 key mass    source-value 26.98154
+   kim_property modify 1 key mass    source-unit amu
 
-where, "species" is a name and "source-value Al" is a map (or a
-key-value pair). For the above example, see
-`Atomic mass <https://openkim.org/properties/show/2016-05-11/brunnels@noreply.openkim.org/atomic-mass>`_
-property definition for more detailed information.
-
-.. note::
-
-    Each map is required to have at least one key-value pair. A pair of
-    "source-value" (a "kaye\_name\_key") and its value
-    (a "key\_name\_value"). Here, the "key\_name\_value" is the value
-    (contents) of the variable. This variable will either be a scalar or an
-    array of specified sextent as defined in the property definition. For
-    example, at the
-    `atomic mass <https://openkim.org/properties/show/2016-05-11/brunnels@noreply.openkim.org/atomic-mass>`_
-    property definition, the "species" key name is a required scalar string
-    variable. In the above example, its "source-value" is a species name,
-    indicated as "Al". The "mass" key name is a required scalar float
-    variable, and its value is an element mass, indicated as "26.98154".
+or, equivalently,
 
-.. note::
+.. parsed-literal::
 
-   `kim-property Python package <https://github.com/openkim/kim-property>`_
-   requires a Python 3.6 or later. For more detailed information, see
-   `KIM package <_kim>`.
+   kim_property create 1 atomic-mass
+   kim_property modify 1 key species source-value Al       &
+                         key mass    source-value 26.98154 &
+                                     source-unit  amu
```

Comment:
I don't think this can work as expected.  I think it needs to be:
```suggestion
   kim_property modify 1 key species source-value Al       &
                         key mass    source-value 26.98154 &
                         key mass    source-unit  amu
```

Reply:
@ellio167 it is working as expected. We do not need to add the key again.